### DISCUSSION TITLE
How to set the rtl of text

### DISCUSSION THREAD

**Discussion by doudouodong** [libvips/libvips#3686] [Q&A]
## Bug report

**Describe the bug**

Most languages ​​are written from left to right, however Arabic is written from right to left。

<!DOCTYPE html>
<html dir=rtl>
<head>
<title>Page Title</title>
</head>
<body>
<h1>×1</h1>
<h1>-SAR20.40</h1>
<h1>30 أغسطس - 10 سبتمبر</h1>
<h1>3 نوفمبر-6 ديسمبر</h1>
<h1>15-24 يناير</h1>
<h1>18 يناير 2023 - 18 يناير 2024</h1>
<h1>18 يناير 2023 - 9 فبراير 2024</h1>
<h1>18-24 يناير 2023</h1>
<h1>ﺮﻘﻣ 23-30</h1>
<h1>23 يوم - 3 ساعات</h1>
<p>This is a paragraph.</p>
</body>
</html>

**Expected behavior**
https://github.com/libvips/libvips/assets/47808515/6abcd8d5-6a9c-4225-8084-d69e34827bab

**Actual behavior**
https://github.com/libvips/libvips/assets/47808515/9b638da0-f107-4977-932f-10c8c8847a0f

**Screenshots**

**Environment**
(please complete the following information)
 - OS: [Ubuntu 16.04, MacOS 10.13]
 - Vips: [ 14.2, etc]

**Additional context**

**Comment by jcupitt** [comment]
Hi @doudouodong,

I don't think pango markup supports this directly. Instead, you can use the unicode text direction characters.
